Packs/PassiveTotal/pack_metadata.json

```diff
@@ -2,10 +2,10 @@
     "name": "PassiveTotal",
     "description": "Analyze and understand threat infrastructure from a variety of sources\u2013passive DNS, active DNS, WHOIS, SSL certificates and more\u2013without devoting resources to time-intensive manual threat research and analysis",
     "support": "partner",
-    "currentVersion": "2.0.2",
+    "currentVersion": "2.0.3",
     "author": "RiskIQ",
     "url": "https://www.riskiq.com",
```

Comment:
@marcellarichmond @mchasepan We need to replace this, what about this one: https://www.riskiq.com/resources/support/

Reply:
approved

Reply:
done